tools/zetasql_helper/zetasql_helper/local_service/local_service.cc

```diff
@@ -33,5 +36,51 @@ absl::Status ZetaSqlHelperLocalServiceImpl::Tokenize(
   return absl::OkStatus();
 }
 
+absl::Status ZetaSqlHelperLocalServiceImpl::ExtractFunctionRange(
+    const ExtractFunctionRangeRequest *request,
```

Comment:
why const pointer? if you don't expect to change the request, we should use const reference.

Reply:
Good point. Changed to reference.